client/components/filters/date/style.scss

```diff
@@ -62,7 +62,7 @@
 	}
 
 	&:focus {
-		outline: 2px solid lightseagreen;
+		box-shadow: inset 0 -1px 0 #00435d, 0 0 0 2px #bfe7f3;
```

Comment:
Where do these colors come from, should we be using [our existing color variables](https://github.com/woocommerce/wc-admin/blob/master/client/stylesheets/abstracts/_colors.scss)?

Reply:
Maybe you can add a comment about this, that it's meant to mirror the GB focused button state? That way we know why it isn't using the color vars.

Reply:
If this color (and its comment) lived in our color variables sheet, it will be easier to surface should someone else need to mimic focus in the future.

Reply:
will make that change.